templates/glue-tables.j2

```diff
@@ -0,0 +1,60 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: The set of Glue tables created
+
+Parameters:
+  Namespace:
+    Type: String
+    Description: >-
+      The namespace string used to build up the path to the correct object keys
+      in the bucket
+    Default: recover
+
+# buckets
+  S3IntermediateBucketName:
+    Type: String
+    Description: Name of the S3 intermediate (JSON) bucket
+
+Resources:
+  # combine all configuration data into a "datasets" collection
+  {% set datasets = [] %}
+  {% for v in sceptre_user_data.dataset_schemas.tables.keys() %}
```

Comment:
Where does this value come from?

Reply:
The sceptre_user_data.dataset_schemas.tables.keys? It comes from here: [config/develop/namespaced/glue-tables.yaml](https://github.com/Sage-Bionetworks/recover/blob/etl-294-create-glue-table-stack/config/develop/namespaced/glue-tables.yaml#L12-L14)

Reply:
But is `dataset_schemas` defined in the `sceptre_user_data`?

Reply:
Yes, it's defined as `src/glue/resources/table_columns.yaml`

Reply:
Ohh, I didn't realize this was set in the tables stack config.